addons/im_livechat/models/discuss_channel.py

```diff
@@ -26,13 +26,25 @@ class DiscussChannel(models.Model):
     chatbot_current_step_id = fields.Many2one('chatbot.script.step', string='Chatbot Current Step')
     chatbot_message_ids = fields.One2many('chatbot.message', 'discuss_channel_id', string='Chatbot Messages')
     country_id = fields.Many2one('res.country', string="Country", help="Country of the visitor of the channel")
+    livechat_failure = fields.Selection(
+        selection=[
+            ("never_answered", "Never Answered"),
+            ("no_one_available", "No one Available"),
+            ("no_failure", "No Failure"),
+        ],
+        string="Live Chat Conversation Failure",
+    )
+    livechat_is_escalated = fields.Boolean("Is livechat escalated", compute="_compute_livechat_is_escalated", store=True)
```

Comment:
Is livechat escalated -> we try to use the term "session" instead of "livechat" in user facing strings